packages/dev/core/src/FlowGraph/Blocks/Execution/Animation/flowGraphPlayAnimationBlock.ts

```diff
@@ -65,8 +76,18 @@ export class FlowGraphPlayAnimationBlock extends FlowGraphAsyncExecutionBlock {
      * @param context
      */
     public _preparePendingTasks(context: FlowGraphContext): void {
-        const targetValue = this.target.getValue(context);
-        const animationValue = this.animation.getValue(context) as Animation;
+        for (const templateStringInput of this.templateStringTargetInputs) {
+            const templateStringValue = templateStringInput.getValue(context);
+            const templateString = templateStringInput.name;
+            this.config.targetPath.setTemplateSubstitution(templateString, templateStringValue);
+        }
+        const targetValue = this.config.targetPath.getProperty(context);

```

Comment:
I like the idea of passing optional path modifiers to the getProperty function. It will simplify future integration of templates to other blocks

Reply:
```typescript
        this.templateTargetComponent.substitutePath(context);
        const targetValue = this.config.targetPath.getProperty(context);
```
When reading these two lines, it's not obvious to understand why it works, because there's no indication that the first line update the same object that the second line query...

I would have added a `getValue()` method in `FlowGraphPathComponent` and embed the call to `substitutePath` inside. It makes things clearer and will prevent a user missing the call to `substitutePath` before getting the value.